crates/ark/src/modules/positron/options.R

```diff
@@ -29,20 +29,19 @@ options(device = function() {
 # Set cran mirror
 repos <- getOption("repos")
 rstudio_cran <- "https://cran.rstudio.com/"
-
 if (is.null(repos) || !is.character(repos)) {
-    options(repos = c(CRAN = rstudio_cran))
+    repos <- c(CRAN = rstudio_cran)
 } else {
     if ("CRAN" %in% names(repos)) {
         if (identical(repos[["CRAN"]], "@CRAN@")) {
             repos[["CRAN"]] <- rstudio_cran
-            options(repos = repos)
         }
     } else {
         repos <- c(CRAN = rstudio_cran, repos)
-        options(repos = repos)
     }
 }
+attr(repos, "Positron") <- TRUE
```

Comment:
I would still be in favor of `IDE` as the attribute name

It's the same idea as when we changed clickable hyperlinks from `rstudio:run` to `x-r-run`, its just more generic and allows other IDEs/editors to opt in without weirdly having to set `"RStudio"` or `"Positron"` as the attribute name.

In other words, to me it is less about getting RStudio to switch to this and more about making it slightly more generic for any IDE/editor to use.